packages/react-components/src/components/SendBox.tsx

```diff
@@ -256,11 +288,24 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     [mergedSendIconStyle, onRenderIcon]
   );
 
+  /* @conditional-compile-remove(file-sharing) */
```

Comment:
Move the conditional compilation directive inside the `useMemo` (and `return ''` in stable build), this way your don't need the conditional compilation directive on line 299 at all. You can *always* have the `SendBoxErrors` component, it just will never be shown in stable build.